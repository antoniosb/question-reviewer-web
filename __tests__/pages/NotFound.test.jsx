import React from 'react'
import NotFoundPage from '../../src/pages/NotFound'

describe('Pages > NotFound', () => {
  test('Render', () => {
    const notFound = shallow(<NotFoundPage />)

    expect(notFound).toMatchSnapshot()
    notFound.find('.not-found-page-button').simulate('click')
  })
})