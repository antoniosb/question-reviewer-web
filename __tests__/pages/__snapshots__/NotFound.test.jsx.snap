// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages > NotFound Render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFoundPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          404
        </h1>,
        <Button
          className="not-found-page-button"
          ghost={false}
          loading={false}
          onClick={[Function]}
          prefixCls="ant-btn"
          size="large"
          type="primary"
        >
          Voltar
        </Button>,
      ],
      "className": "not-found-page",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "404",
        },
        "ref": null,
        "rendered": "404",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Voltar",
          "className": "not-found-page-button",
          "ghost": false,
          "loading": false,
          "onClick": [Function],
          "prefixCls": "ant-btn",
          "size": "large",
          "type": "primary",
        },
        "ref": null,
        "rendered": "Voltar",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            404
          </h1>,
          <Button
            className="not-found-page-button"
            ghost={false}
            loading={false}
            onClick={[Function]}
            prefixCls="ant-btn"
            size="large"
            type="primary"
          >
            Voltar
          </Button>,
        ],
        "className": "not-found-page",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "404",
          },
          "ref": null,
          "rendered": "404",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Voltar",
            "className": "not-found-page-button",
            "ghost": false,
            "loading": false,
            "onClick": [Function],
            "prefixCls": "ant-btn",
            "size": "large",
            "type": "primary",
          },
          "ref": null,
          "rendered": "Voltar",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
